core-bundle/src/HttpKernel/JwtManager.php

```diff
@@ -81,7 +81,7 @@ public function parseRequest(Request $request): ?array
             }
         }
 
-        if ('/contao/login' === $request->getPathInfo()) {
+        if (0 === strpos($request->getPathInfo(), '/contao/')) {
```

Comment:
This feels strange to me...what if somebody adjusts the paths of the routes? Sounds like a task for a request matcher here maybe?

Reply:
This is used in the entry point script. There is not container, no kernel, no nothing. I had to extend the request match to allow the logout and two-factor routes, otherwise the 2fa process cannot be cancelled or configured (if enforced) when accessing the backend using `preview.php`. I think we can safely assume the back end routes are correctly protected, because the access control list already does that: https://github.com/contao/contao/blob/master/manager-bundle/src/Resources/skeleton/app/security.yml#L80